packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -318,11 +377,45 @@ class ImageCache {
     return result;
   }
 
+  /// The [ImageCacheStatus] information for the given `key`.
+  ImageCacheStatus statusForKey(Object key) {
+    return ImageCacheStatus._(
+      pending: _pendingImages.containsKey(key),
+      keepAlive: _cache.containsKey(key),
+      live: _liveImages.containsKey(key),
+    );
+  }
+
   /// Returns whether this `key` has been previously added by [putIfAbsent].
   bool containsKey(Object key) {
     return _pendingImages[key] != null || _cache[key] != null;
   }
 
+  /// The number of live images being held by the [ImageCache].
+  ///
+  /// Compare with [ImageCache.currentSize] for completed held images.
```

Comment:
Not sure what "completed held images" is?

Reply:
Missed a text replacement.  done.